src/App.vue

```diff
@@ -176,7 +176,14 @@
       <v-content>
         <router-view />
       </v-content>
-      <v-footer app></v-footer>
+      <v-footer app justify-center dark color="primary" height="auto">
```

Comment:
I think you don't need to add the dark attribute here. As far as I understand it is only supposed to be applied when the app has a defined dark theme.

https://vuetifyjs.com/en/framework/theme#light-and-dark

Reply:
The toolbar is also using `dark` https://github.com/DD-DeCaF/caffeine-vue/blob/697d999f31a047e93e721f20f21fe695de72fb14/src/App.vue#L4 otherwise the font is black. I also feel like it should be the job of vuetify to determine a font color that has enough contrast but it doesn't.

Reply:
When @kvikshaug set up the skeleton app we didn't spend a lot of time understanding the theming options in Vuetify. I'd be keen revisit that once we've progressed to a point where the vue rebuild is legit :D

I think it is OK to keep the `dark` for now but, if we wanted solid theming we have to do much of it ourselves. Vue's defaults are rudimentary at best and after some testing, I can't really say that it automatically sets the font color for you.